documentation/versioned_docs/version-2.x/examples/geomaptool.mdx

```diff
@@ -0,0 +1,107 @@
+- 'APIKey': API access token of Mapbox.
+- 'id': Map style to be used.
+  - MapboxStyle.Streets, MapboxStyle.Outdoor, MapboxStyle.Light, MapboxStyle.Dark, MapboxStyle.Satellite,
+MapboxStyle.Satellite_Streets, MapboxStyle.Navigation_Day, MapboxStyle.Navigation_Night, MapboxStyle.Navigation_Guide_Day, MapboxStyle.Navigation_Guide_Night.
+- 'tileFormat': Map image tile format.
+  - MapboxImageFormat.PNG, MapboxImageFormat.PNG32, MapboxImageFormat.PNG64, MapboxImageFormat.PNG128,
+  MapboxImageFormat.PNG256, MapboxImageFormat.JPG70, MapboxImageFormat.JPG80, MapboxImageFormat.JPG90, MapboxImageFormat.PNGRAW
+- 'latlong': Latitude & Longitude location where the model will be placed on the map.
+
+```jsx runnable
+// import { GeomapTool, MapProviders, MapboxMode, MapboxStyle, MapboxImageFormat } from '@cognite/reveal/tools';
+
+const mapConfig = {
+  provider: MapProviders.MapboxMap,
+  mode: MapboxMode.Style,
+  APIKey: "pk.eyJ1IjoidGVudG9uZSIsImEiOiJjazBwNHU4eDQwZzE4M2VzOGhibWY5NXo5In0.8xpF1DEcT6Y4000vNhjj1g",
+  id: MapboxStyle.Satellite_Streets,
+  tileFormat: MapboxImageFormat.JPG70,
+  latlong: {
+      latitude: 59.9016426931744,
+      longitude: 10.607235872426175
+  }
+}
+new GeomapTool(viewer, mapConfig);;
+```
+## Bing Maps
+Configuration of the Bing Maps are listed below
+- 'APIKey': API access token of Bing Map.
+- 'type': The type of the map used.
+  - BingMapType.Aerial, BingMapType.Road, BingMapType.Aerial_Labels, BingMapType.Oblique, BingMapType.Oblique_Labels.
+- 'latlong': Latitude & Longitude location where the model will be placed on the map.
+
+```jsx runnable
+// import { GeomapTool, MapProviders, BingMapType } from '@cognite/reveal/tools';
+
+const mapConfig = {
+  provider: MapProviders.BingMap,
+  APIKey: "AuViYD_FXGfc3dxc0pNa8ZEJxyZyPq1lwOLPCOydV3f0tlEVH-HKMgxZ9ilcRj-T",
+  type: BingMapType.Aerial,
+  latlong: {
+      latitude: 59.9016426931744,
+      longitude: 10.607235872426175
+  }
+}
+
+new GeomapTool(viewer, mapConfig);
+```
+
+## Here Maps
+Configuration of the Here Maps are listed below
+- 'APIKey': API access token of Here Map.
+- 'appCode': Application code token of Here Map.
+- 'style': The type of maps to be used.
+  - HereMapType.Aerial, HereMapType.Base, HereMapType.Pano, HereMapType.Traffic
+- 'scheme': Specifies the view scheme. Be aware that invalid combinations of schemes and tiles are rejected. For all satellite,
+ hybrid and terrain schemes, you need to use the Aerial Tiles base URL instead of the normal one.
+  - HereMapScheme.Day, HereMapScheme.Night, HereMapScheme.Terrain, HereMapScheme.Satellite.
+- 'imageFormat': Map image format.
+  - HereMapImageFormat.PNG, HereMapImageFormat.PNG8, HereMapImageFormat.JPG.
+
+```jsx runnable
+// import { GeomapTool, MapProviders, HereMapType, HereMapScheme, HereMapImageFormat } from '@cognite/reveal/tools';
+
+const mapConfig = {
+  provider: MapProviders.HereMap,
+  APIKey: "HqSchC7XT2PA9qCfxzFq",
+  appCode: "5rob9QcZ70J-m18Er8-rIA",
```

Comment:
Can you hide the API-keys in customScope.ts? So you define e.g. `HereMapApiKey` an `HereMapAppCode` in customScope.ts (and similar for the other examples) and use this constant instead? See https://github.com/cognitedata/reveal/blob/945c4d190707793f769e370e50a3ab8d75372e49/documentation/versioned_docs/version-2.x/examples/cad-3dobjects.mdx#L64-L64 for a similar example.

This makes finding the API keys harder which will reduce chance of people just copying these.